test/powershell/scenarios.json

```diff
@@ -3,7 +3,8 @@
     "image": "mcr.microsoft.com/devcontainers/base:jammy",
     "features": {
       "powershell": {
-        "modules": "az.resources, az.storage"
+        "modules": "az.resources, az.storage",
+        "powershellProfileURL": "https://raw.githubusercontent.com/nebula-it/features/patch-2/test/powershell/testProfile.ps1"
```

Comment:
Would you be able to support and maintain it for longer term? I am worried that the test might break in future if that's not the case.

Can we use https://github.com/codspace/powershell-profile/blob/main/Test-Profile.ps1 instead? Updated the contents to match your test file

Reply:
Yeap I can keep it, but using an external repo does not seem like a good solution to me.
So my thought process was that once this was merged, we can point to https://github.com/devcontainers/features/blob/main/test/powershell/testProfile.ps1 so its in same repo

Reply:
Otherwise I can point it to your repo above, let me know which makes more sense to you

Reply:
Oh I see that, above repo you posted is not a personal one. I have updated code to use that.